src/ResourceManager/Profile/Commands.Profile.Test/AzureRMProfileTests.cs

```diff
@@ -75,7 +77,7 @@ public void MultipleTenantsAndSubscriptionsSucceed()
             tenantResults = client.ListTenants(DefaultTenant.ToString());
             Assert.Equal(1, tenantResults.Count());
             AzureSubscription subValue;
-            Assert.True(client.TryGetSubscription(DefaultTenant.ToString(), DefaultSubscription.ToString(), out subValue));
```

Comment:
Can we also cover GetSUbscriptionByName in this test method, or add another unit test for this?